frontend/src/components/CardDetailsPage.tsx

```
@@ -66,6 +80,40 @@ const DetailsCard = ({
+                    }}
+                    children="Edit Program"
+                  />
+
+                  <ActionButton
+                    children=" Add Module"
+                    onClick={() => {
+                      router.push(`${window.location.pathname}/createModule`)
+                    }}
+                  />
+                  {isDraft && (
+                    <ActionButton children="Publish" onClick={() => setPublish && setPublish()} />
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)